pom.xml

```diff
@@ -465,25 +466,32 @@
         <scope>provided</scope>
       </dependency>
 
+      <!-- Avro 1.8.2 includes Paranamer 2.7, but Spark 2.4 needs Paranamer 2.8, forcing that version -->
+      <dependency>
```

Comment:
Is this a runtime dependency? if so, can this be isolated to the spark and utilities bundles?

Reply:
+1